habana/1.13.0/ubi8-python-3.8/Pipfile

```diff
@@ -54,6 +54,11 @@ jupyterlab-widgets = "~=3.0.10"
 jupyter-resource-usage = "~=0.7.2"
 nbdime = "~=3.2.1"
 nbgitpuller = "~=1.2.0"
+# pycodestyle is dependency of below packages 
+# and to achieve compatible of pycodestyle with python-lsp-server[all]
+# pinned the below packages
+autopep8 = "~=2.0.4"
+flake8 = "~=7.0.0"
```

Comment:
could these problems be caused by locking depenencies in a python > 3.8? https://github.com/opendatahub-io/notebooks/issues/574

Reply:
I believe we should start to offboard the UBI8 runtimes since we don't use them at all in the project. They introduce a lot of unnecessary maintenance.

Reply:
I dont think it the problem due to python version, more of dependency resolution of pipenv
@jiridanek 

i agree @atheo89 , we should.